sampleconfig/core.yaml

```diff
@@ -454,13 +454,11 @@ peer:
     #     library: /etc/hyperledger/fabric/plugin/escc.so
     handlers:
         authFilters:
-          -
-            name: DefaultAuth
```

Comment:
Looking at this, I'm not even sure we should have activated the expiration check by default.
So, I am not sure we should have this new check activated by default.

Perhaps let it be commented out?

Reply:
1. the DefaultAuth and ExpirationCheck filters were enabled by default. I don't know how long it's been on.

2. I think that enabling filters by default makes the factory safer for newcomers who are just starting to use and familiarize themselves with it.

3. Confident professionals can comment these filters consciously, realizing what it can lead to.

Reply:
In that case I'd like to hear other people's thoughts on this. 

@C0rWin , or @denyeart  wdyt ?

Reply:
I think this is a great feature for v3, thank you @pfi79.
Since it is a major release, I think it is completely acceptable to add a new check, as long as we remember to mention it in the release notes.

Reply:
I think I do agree with @denyeart. This is a good feature as it will eliminate the issues with newcomers, and given v3.0 already brings some breaking changes, IMO,  this should not be a problem enabling it.